raft/rafttest/network.go

```diff
@@ -59,6 +70,18 @@ func (rn *raftNetwork) delay(from, to uint64, d time.Duration, rate float64) {
 	panic("unimplemented")
 }
 
+func (rn *raftNetwork) disconnect(id uint64) {
+	rn.mu.Lock()
+	defer rn.mu.Unlock()
+	rn.recvQueues[id] = nil
```

Comment:
ah... I was trying to do with `drop` but it will panic at the moment... Should add TODO. 

But after awhile, I found it would not be a good solution since setting `drop` affects the previous `drop` setting.

I will solve it in this commit later.